### ISSUE TITLE
missing and incorrect genes associated in fanconi disease group

### ISSUE THREAD

**Issue by dnahotline** [monarch-initiative/monarch-disease-ontology-RETIRED#167]
Three newer fanconi complementation groups are missing from this [page](https://monarchinitiative.org/disease/DOID:13636#overview)
FANCU	(XRCC2)
FANCR	(RAD51)
FANCV	(REV7)

Also three genes that were listed should not have been included
FANCD2OS is not the right terminology
MX1 and	TNF are not fanconi genes

I'll follow up with pub med references for the new ones
 mucho help from @jmcmurry on this my first ticket!!!

**Comment by jmcmurry** [comment]
There's also something odd about the gene to the number of gene-disease relations. Even after the associations above are fixed, there seems to be something wacky with the reasoning that the defacto parent disease doesn't have each of the child associations. Here are the results pivoted for clarity.

<img width="1056" alt="screen shot 2017-01-14 at 6 22 20 pm" src="https://cloud.githubusercontent.com/assets/3658863/21959750/503990a6-da86-11e6-856c-c85992469b46.png">

**Comment by dnahotline** [comment]
@jmcmurry i'm not used to looking at this. what do the 1's stand for in the columns and rows. BRCA2 has a 1 for BRCA2 but also for FANCA.

**Comment by dnahotline** [comment]
@jmcmurry also RAD51 FANCR is not on the list of FA genes and should be.

**Comment by dnahotline** [comment]
notes 
* Mutation in RAD51C is associated with a Fanconi anemia-like syndrome (FANCO) (OMIM: 613390)
**specific mutations in ERCC4 (XPF) are associated with xeroderma pigmentosum (OMIM: 278760) and XFE progeroid syndrome  (OMIM: 610965)  or FA (FANCQ)
***A dominant mutation in RAD51 (FANCR) is associated with a Fanconi anemia-like syndrome

**Comment by mellybelly** [comment]
Also had noticed we have one disease group "Fanconi's Anemia"
https://monarchinitiative.org/disease/DOID:13636
that should have the label changed to Fanconi Anemia.
Will make a ticket on the DO tracker for this one, but should make sure the capitalization/form is same for the whole collection.

**Comment by cmungall** [comment]
Was fixed #227 

Unfortunately not yet in latest mondo

**Comment by cmungall** [comment]
Missing links to the grouping class: https://github.com/DiseaseOntology/HumanDiseaseOntology/issues/250

**Comment by mellybelly** [comment]
what is the status of this? need to review all Fanconi subtypes and genes and phenotypes to ensure up to date. HIGH PRIORITY, Perhaps @nicolevasilevsky can work with @dnahotline.

**Comment by nicolevasilevsky** [comment]
@mellybelly let me know how I can help

**Comment by mellybelly** [comment]
we need a plan for how to address this asap. It involves fixes to Mondo and fixes to the annotations. Where should this be done? HPOA, as well as some boutique disease-gene/variant associations? Can we set up INCA for this or ???? @cmungall please advise. Also @pnrobinson

**Comment by mellybelly** [comment]
I think this will require some NER in combination with id mapping. Perhaps @TomConlin and @putmantime can assist?

**Comment by pnrobinson** [comment]
AS I recall, Fanconi was one of those diseases where people published the biochemistry in Nature Genetics but nobody published detailed phenotype analyses for the various subtypes.

Maureen, what is the best place to get annotation data for Fanconi?

-Peter


Peter Robinson

Professor and Donald A. Roux Chair, Genomics and Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

New Book: Computational Exome and Genome Analysis<https://www.crcpress.com/Computational-Exome-and-Genome-Analysis/Robinson-Piro-Jager/p/book/9781498775984>

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by pnrobinson** [comment]
Blanche alter and Arleen auerbach  have the most phenotype data.
There are paper in our shared doc paperpile.
I can send later.
Seems like someone who knows really knows data and ontology etc should go to the Fa patient meeting and catalog. I’ve gotten me and Melissa invited. Perhaps Peter You would like to attend?



Sent from my iPhone

On Oct 31, 2017, at 1:44 PM, Peter Robinson <Peter.Robinson@jax.org<mailto:Peter.Robinson@jax.org>> wrote:


AS I recall, Fanconi was one of those diseases where people published the biochemistry in Nature Genetics but nobody published detailed phenotype analyses for the various subtypes.

Maureen, what is the best place to get annotation data for Fanconi?

-Peter


Peter Robinson

Professor and Donald A. Roux Chair, Genomics and Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org<http://www.jax.org>

Robinson lab: https://robinsongroup.github.io/

New Book: Computational Exome and Genome Analysis<https://www.crcpress.com/Computational-Exome-and-Genome-Analysis/Robinson-Piro-Jager/p/book/9781498775984>

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by nicolevasilevsky** [comment]
This issue was moved to monarch-initiative/mondo#187
